modules/currency-l1/src/main/scala/io/constellationnetwork/currency/l1/CurrencyL1App.scala

```diff
@@ -77,11 +77,18 @@ abstract class CurrencyL1App(
   val kryoRegistrar: Map[Class[_], KryoRegistrationId[KryoRegistrationIdRange]] =
     nodeSharedKryoRegistrar.union(dagL1KryoRegistrar)
 
+  val networkStateAfterJoining: NodeState = NodeState.Ready
+
   def run(method: Run, nodeShared: NodeShared[IO, Run]): Resource[IO, Unit] = {
     import nodeShared._
 
     for {
-      cfgR <- ConfigSource.default.loadF[IO, AppConfigReader]().asResource
+      cfgR <- ConfigSource
+        .resources("currency-l1.conf")
+        .withFallback(ConfigSource.resources("dag-l1.conf"))
```

Comment:
@marcinwadon this was a consequence of the assembly merging the `application.conf` files. I could replicate the settings from the `dag-l1.conf` into `currency-l1.conf` but went with this approach instead so that any changes to parameters within `dag-l1.conf` would also propagate to CurrencyL1. This is more closely aligned with the previous behavior rather than duplicating the parameters between the two configs.